### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Typos and general tidy.

### CODE REVIEW

File: site/content/integrate/apps/_index.md [hunk 1]
```diff
@@ -41,20 +41,20 @@ Read the [JavaScript start guide]({{< ref "quick-start-js" >}}) or [Go quick sta
...
 
-A plugin should be used when you need to alter the React UX in Mattermost or have an application that requires low-latency with the server (such as replacing characters in any message). Currently, plugins have several UX hooks that Apps cannot access, however we plan to add/migrate more UX hooks into the Apps Framework over time.
+A plugin should be used when you need to alter the UX in Mattermost or have an application that requires low-latency with the server (such as replacing characters in any message). Currently, plugins have several UX hooks that apps cannot access, however we plan to add/migrate more UX hooks into the Apps Framework over time.
 
 ### What's the difference between the apps framework and the plugin framework?
 
-The App framework provides a few differences from plugins.
+The app framework provides a few differences from plugins.
```
Comment (hanzei) [inline site/content/integrate/apps/_index.md:52]: Didn't we agree one
```suggestion
The Apps Framework provides a few differences from plugins.
```
?

**Review by hanzei — APPROVED** [review]
LGTM

File: site/content/integrate/apps/_index.md (hunk 1 above)
Comment (aaronrothschild) [reply site/content/integrate/apps/_index.md:52]: @hanzei I wish there was a linter for Docs. ;)

**Review by aaronrothschild — APPROVED** [review]

File: site/content/integrate/apps/_index.md (hunk 1 above)
Comment (mickmister) [reply site/content/integrate/apps/_index.md:52]: I agree. 1/5 "Apps Framework" seems right as a proper noun

**Pull request merged**